### PR TITLE
Fixed the Prover installation script for Windows (Korean)

### PR DESCRIPTION
**Pull request by junkil-park**
### Description
* Fixed the Prover installation script for Windows (Korean language)
* Upgraded the boogie version

### CODE REVIEW

File: scripts/windows_dev_setup.ps1 [hunk 1]
```diff
@@ -409,15 +409,15 @@ function install_dotnet {
     Write-Host "Installing Microsoft DotNet..."
     winget install "Microsoft.DotNet.SDK.$dotnet_version" --accept-source-agreements --silent
 
-    $dotnet_version = winget show -e "Microsoft.DotNet.SDK.$dotnet_version" | Select-String Version
+    $dotnet_version = winget show -e "Microsoft.DotNet.SDK.$dotnet_version" | Select-String -pattern "Version|버전"
```
Comment (banool) [inline scripts/windows_dev_setup.ps1:412]: Does winget have an option to output this information in a programmatic way? For example output the data as JSON + where the keys don't change based on the language. As far as I can tell this is just grepping the standard output right now.

**Review by rahxephon89 — APPROVED** [review]

**Review by aalok-t — APPROVED** [review]

**Comment by wrwg** [comment]
@aalok-t you accidentally added some commit here? Be careful to not merge this!

**Review by wrwg — DISMISSED** [review]
What is @aalok-t commit doing in this PR? Blocking merge until this is clarified.

**Comment by rahxephon89** [comment]
> @aalok-t you accidentally added some commit here? Be careful to not merge this!

Yeah,  but it seems to be an empty one

**Comment by wrwg** [comment]
> > @aalok-t you accidentally added some commit here? Be careful to not merge this!
> 
> Yeah, but it seems to be an empty one

Please remove this commit by force pushing, @junkil-park

File: scripts/windows_dev_setup.ps1 (hunk 1 above)
Comment (junkil-park) [reply scripts/windows_dev_setup.ps1:412]: I tried with different values for `--locale` (https://learn.microsoft.com/en-us/windows/package-manager/winget/show), but there wasn't any luck. It seems that `winget show -e ...` displays keys and values in the standard output, where the keys is in Korean if the Windows system language is Korean.

**Comment by junkil-park** [comment]
> > > @aalok-t you accidentally added some commit here? Be careful to not merge this!
> > 
> > 
> > Yeah, but it seems to be an empty one
> 
> Please remove this commit by force pushing, @junkil-park

Done force pushing. (@wrwg )

File: scripts/windows_dev_setup.ps1 (hunk 1 above)
Comment (gregnazario) [reply scripts/windows_dev_setup.ps1:412]: We will want to also handle Chinese and others then in the future... We will need to think about if we can either programmatically do it or convert the input to English to be consistent.

**Review by gregnazario — APPROVED** [review]

**Pull request merged**